.github/workflows/pulp_images.yml

```diff
@@ -240,11 +240,11 @@ jobs:
         run: |
           if [ "${{ matrix.app.s6 }}" == "true" ]; then
             # Reuse the folders from the s6 mode tests
-            FILE="docker-compose.folders.yml"
+            FILE="compose.folders.yml"
             # We'll pull the web image from a registry since we didn't build it.
             WEB_TAG="${APP_BRANCH}"
           else
-            FILE="docker-compose.yml"
+            FILE="compose.yml"
```

Comment:
I was looking at the compose spec, and I see that docker-compose now prefers compose.yaml or compose.yml . Nice!